subsys/bluetooth/shell/bt.c

```diff
@@ -145,6 +155,15 @@ static struct bt_scan_filter {
 	bool pa_interval_set;
 } scan_filter;
 
+static bool scan_verbose_output = false;
+static char *scan_response_type_txt[] = {
```

Comment:
These should probably be moved into the `#if defined(CONFIG_BT_OBSERVER)` clause

Reply:
I believe they already are?